README.md

```diff
@@ -61,6 +61,21 @@ Para realizar la ejecución de los test se debe utilizar la orden:
 make test
 ```
 
+## Elección del contenedor para automatizar las pruebas
+
+En primer lugar, debemos elegir la imagen base para el contenedor que automatizará las pruebas. 
+Los criterios serán principalmente lo ligera que sea la imagen y que la versión del lenguaje Ruby sea la última.
+
+La versión con la que trabajaremos será la última actualmente, que es la 3.2. Siguiendo también las [buenas prácticas a la hora de crear un contenedor para Ruby](https://lipanski.com/posts/dockerfile-ruby-best-practices), debemos usar solamente versiones oficiales o versiones propias, por lo tanto he mirado en la [página oficial](https://hub.docker.com/_/ruby), las imágenes base disponibles para esta versión.
```

Comment:
No hay ninguna razón específica para usar versiones oficiales.

Reply:
En la guía de buenas prácticas que sigo dice que siempre se deben usar versiones oficiales.

Reply:
Ya te dije que la guía que usas es para contenedores en producción, *no de prueba*. En clase os expliqué explícitamente todos los tipos de bases que había y qué tenían de positivo cada uno para pruebas, y también para aprender.
En todo caso, los comentarios que os hago tratan de que avancéis en el objetivo de aprendizaje. Si no te son útiles, que los contradigas tampoco es útil ni para ti ni para mi. Simplemente no les hagas caso en la siguiente versión que envíes, y yo trataré, como siempre, de evaluar si has alcanzado los objetivos de aprendizaje o no.